Bluetooth: BAP: Shell: Improve recv statistics

Thalley opened the pull request:
Add more information when we print the recv every 100th packet, and remove all per-recv printing.

This also resets all information on stream start. This does, however, no properly support multiple streams.

Thalley committed:
Bluetooth: BAP: Shell: Improve recv statistics

Add more information when we print the recv every 100th
packet, and remove all per-recv printing.

This also resets all information on stream start. This does,
however, no properly support multiple streams.

Signed-off-by: Emil Gydesen <emil.gydesen@nordicsemi.no>

larsgk commented:
We should keep track of all the places that needs to be changed for multi stream support ;)

larsgk commented on subsys/bluetooth/audio/shell/bap.c:
```diff
@@ -1637,29 +1637,41 @@ static struct bt_bap_broadcast_sink_cb sink_cbs = {
+		lost_pkts++;
+	}
+
+	/* TODO: Make it possible to only print every X packets, and make X settable by the shell */
```
Sounds like a good idea to make it configurable.  Also it seems like a fairly small change.  Could this be added in the PR? otherwise LGTM

Thalley replied:
I think it makes sense to add to this PR :)

Thalley committed:
Bluetooth: BAP: Shell: Add runtime config of recv_stats_interval

The internval of how often we report
receive stats can now be configured via the cmd_recv_stats
command.

Signed-off-by: Emil Gydesen <emil.gydesen@nordicsemi.no>

Thalley commented:
> We should keep track of all the places that needs to be changed for multi stream support ;)

Yeah, it's getting increasingly difficult :)
